scene/gui/tree.cpp

```diff
@@ -4190,6 +4196,12 @@ void Tree::_determine_hovered_item() {
 	}
 }
 
+void Tree::_on_dropping_unfold_timer_timeout() {
+	if (drop_mode_over && drop_mode_section == 0 && drop_mode_over->is_collapsed()) {
```

Comment:
```suggestion
	if (drop_mode_over && drop_mode_section == 0) {
```
`set_collapsed()` already checks this.